models/error.go

```diff
@@ -1371,6 +1372,48 @@ func (err ErrMergePushOutOfDate) Error() string {
 	return fmt.Sprintf("Merge PushOutOfDate Error: %v: %s\n%s", err.Err, err.StdErr, err.StdOut)
 }
 
+// ErrPushRejected represents an error if merging fails due to unrelated histories
```

Comment:
However it's not always due to unrelated histories? (e.g. hook is failing)

Reply:
That's a copy and paste error.

Reply:
done